tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/InstrumentationTests.cs

```diff
@@ -152,7 +157,7 @@ public async Task WhenUsingPathWithDotsInInTracerHome_InstrumentsApp()
         {
             SetEnvironmentVariable("DD_INTERNAL_WAIT_FOR_DEBUGGER_ATTACH", "1");
             // not using Path.Combine here because it resolves the .. and we want it in the path
-            var path = EnvironmentHelper.MonitoringHome + "/../" + Path.GetFileName(EnvironmentHelper.MonitoringHome);
+            var path = Path.Combine(EnvironmentHelper.MonitoringHome, "..", Path.GetFileName(EnvironmentHelper.MonitoringHome)!);
```

Comment:
I did the concatenation manually on purpose, and added the comment above :p
if you look at the resulting path here, it has no `..` in it, kinda defeating the purpose of the test.

Reply:
🤔 It doesn't resolve it? e.g.

```
Console.WriteLine(Path.Combine(Environment.CurrentDirectory, "..", "somepath.exe"));
```

gives

```
/app/../somepath.exe
```

(just tested it) 🤔 

And similarly, in the tracer logs, I see:

```
Resolving managed profiler directory to: C:\repos\dd-trace-dotnet-3\shared\bin\monitoring-home\..\monitoring-home\net6.0
```

which shows that it _is_ there?

Reply:
ok, yeah, right, I must have been at low brain % yesterday. I removed the comment.